lib/index.js

```diff
@@ -62,7 +62,7 @@ exports.register = function githubauth (server, options, next) {
   server.route([
     {
       method: 'GET', // ping
-      path: OPTIONS.REDIRECT_URL, // must be identical to Authorized callback URL
+      path: process.env.REDIRECT_URL, // must be identical to Authorized callback URL
```

Comment:
I would _default_ this to `/githubauth` so people don't _have_ to set it but have the _option_ of setting it if they _want_ to... 💭